tensorrt_llm/_torch/modules/gated_mlp.py

```diff
@@ -75,13 +74,12 @@ def __init__(self,
             self.hidden_size,
             bias=bias,
             dtype=dtype,
-            parallel_config=ParallelConfig(
-                tensor_parallel_rank=tp_rank,
-                tensor_parallel_size=tp_size,
-                tensor_parallel_mode=TensorParallelMode.ROW,
-                gpus_per_node=gpus_per_node,
-                pipeline_parallel_size=pp_size,
-                parallel_rank=config.mapping.rank),
+            mapping=Mapping(world_size=tp_size * pp_size,
```

Comment:
This mapping is the same as the one on line 60. You can create one Mapping object and use it in both places.

Reply:
Done